libs/providers/flipt/README.md

```diff
@@ -7,7 +7,7 @@ This provider is an implementation on top of the official [Flipt Node Server Sid
 ## Installation
 
 ```
-$ npm install @openfeature/flipt-provider
+npm install @openfeature/flipt-provider
```

Comment:
@markphelps because you made these small changes in the `/providers/flipt` folder, our release automation will also add this PR to to the original flipt provider's changelog, and increment the it's version number as if a new feature was added. In short, you will see: `feat: Add Flipt web provider` in your server side provider's release-notes/changelog. This isn't a deal-breaker for me, I think most people will just ignore it, but the alternative is to do these changes in a separate PR.

Reply:
thanks! i reset those to origin and will do in another PR